app/styles/ui/banners/_successful.scss

```diff
@@ -1,4 +1,5 @@
-#successful {
+#successful,
+#successful-merge {
```

Comment:
This fixes the styling issue of the "branch updated" banner for merges, which didn't show the green circle with the checkmark inside.